docs/user/status.md

```diff
@@ -8,6 +8,7 @@ This document describes those conditions and, where appropriate, suggests possib
 When `RetrievedUpdates` is `True`, the CVO is succesfully retrieving updates, which is good.
 When `RetrievedUpdates` is `False`, `reason` will be set to explain why, as discussed in the following subsections.
 In all cases, the impact is that the cluster will not be able to retrieve recommended updates, so cluster admins will need to monitor for available updates on their own or risk falling behind on security or other bugfixes.
+When CVO is unable to retrieve recommended updates the ClusterVersionOperatorCannotRetrieveUpdates alert will fire containing the reason.
```

Comment:
IMO we should add that for `NoChannel` we will not alert or whatever the final conclusion on this.